### PR TITLE
Result statistical data

### PR DESCRIPTION
**Pull request by josef-janousek**

### CODE REVIEW

File: odalic/src/main/java/cz/cuni/mff/xrg/odalic/tasks/annotations/StatisticalAnnotation.java [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+ * Annotates column in a table for statistical data processing.
+ * 
+ * @author Josef Janoušek
+ *
+ */
+@Immutable
+@XmlJavaTypeAdapter(StatisticalAnnotationAdapter.class)
+public final class StatisticalAnnotation {
+
+  private final Map<KnowledgeBase, ComponentTypeValue> component;
```
Comment (brodecva) [inline odalic/src/main/java/cz/cuni/mff/xrg/odalic/tasks/annotations/StatisticalAnnotation.java]: @josef-janousek Is this right? Why is there a map from KBs and not from columns? The same for the predicates...

Comment (brodecva) [inline odalic/src/main/java/cz/cuni/mff/xrg/odalic/tasks/annotations/StatisticalAnnotation.java]: @josef-janousek Oh, nevermind. I got it. It is indexed by a column index, right?

Comment (josef-janousek) [inline odalic/src/main/java/cz/cuni/mff/xrg/odalic/tasks/annotations/StatisticalAnnotation.java]: @brodecva Yes, the same way as for HeaderAnnotation.

Comment (josef-janousek) [inline odalic/src/main/java/cz/cuni/mff/xrg/odalic/tasks/annotations/StatisticalAnnotation.java]: @brodecva Yes, the same way as for HeaderAnnotation.

**Pull request merged**